dask/array/overlap.py

```diff
@@ -637,6 +637,14 @@ def map_overlap(
         trim = get(sig.index("trim"), args, trim)
         func, args = args[0], [func]
 
+    if boundary is None:
+        # Default boundary value is set in the function named "coerce_boundary"
+        warnings.warn(
+            "Default 'boundary' argument value will change from 'reflect' "
+            "to 'none' in future versions.",
```

Comment:
I often see things like, "use `boundary="none"` to opt into future behavior now". When we flip the default in the future, we can add a note in the docstring as well.

Reply:
Sounds like a nice extra detail - added now.